src/app/docs/summary-action-bar/summary-action-bar-docs.component.html

```diff
@@ -0,0 +1,135 @@
+<sky-docs-demo-page
+  moduleName="SummaryActionBarModule"
+  moduleSourceCodePath="src/app/public/modules/summary-action-bar/"
+  pageTitle="Summary action bar"
+>
+
+<sky-docs-demo-page-summary>
+  The summary action bar module provides a docked container for actions and summary information on forms or multiselect lists.
+</sky-docs-demo-page-summary>
+
+  <sky-docs-design-guidelines>
+    <sky-docs-demo-page-section
+      heading="Usage"
+    >
+      <sky-docs-design-guideline
+        heading="Use when"
+        headingStyle="success"
+      >
+        <p class="sky-docs-text-block">
+          Use summary action bars to hold actions and summary information related to the parent container to which it is docked.
+        </p>
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline>
+        <p class="sky-docs-text-block">
+          Use summary action bars for actions that can be taken on multiple items in a list of items. When used with a multiselect list, hide the summary action bar until users select an item in the list. After users select an item, the summary action bar should fly in from the bottom of the page.
+        </p>
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline>
+        <p class="sky-docs-text-block">
+          Use summary action bars for actions that are taken on the selected record in the split view.
+        </p>
+      </sky-docs-design-guideline>
+      <sky-docs-design-guideline>
+        <p class="sky-docs-text-block">
+          Use summary actio bars when you need to show summary information related to the form in a modal.
+        </p>
+      </sky-docs-design-guideline>
+
+      <sky-docs-design-guideline
+        heading="Don't use when"
+        headingStyle="danger"
+      >
+        <p class="sky-docs-text-block">
+          Don't use summary action bars to wrap a large amount of content will eat into the vertical space on the page. Show the least amount of information that the user needs to inform their actions. Explore alternate layouts if you need to show a large amount of summary information.
```

Comment:
```suggestion
          Don't use summary action bars to wrap a large amount of content because it eats into the vertical space on the page. Show the least amount of information that users need to inform their actions. Explore alternate layouts if you need to show a large amount of summary information.
```